adapters/Vungle/VungleAdapter/GADMAdapterVungleBanner.m

```diff
@@ -213,4 +213,8 @@ - (void)willLeaveApplication {
   [_connector adapterWillLeaveApplication:_adapter];
 }
 
+- (void)rewardUser {
+  NSLog(@"%@ received rewardUser event.", NSStringFromClass(self.class));
```

Comment:
This can just be:

```
// Do nothing.
```

I don't want to confuse users if this log were to make it through.

Reply:
@ericleich Updated, thanks a lot for the guidance. ^_^